<?xml version="1.0"?>
<launch>
  <arg name="mycobot_port" default="/dev/ttyUSB0"/>
  <arg name="urdf_path" default="$(find mycobot_moveit)/urdf/mycobot_urdf_gazebo.urdf"/>

  <!-- send robot urdf to param server -->
  <param name="robot_description" textfile="$(arg urdf_path)" />


  <!-- myCobot -->
  <node name="mycobot_control" pkg="mycobot_moveit" type="mycobot_control" output="screen" respawn="false" />
  <!-- HW Interface -->
  <node name="hardware_interface" pkg="mycobot_moveit" type="mycobot_hw_interface.py" output="screen">
    <param name="mycobot_port" value="$(arg mycobot_port)"/>
  </node>

  <include file="$(find mycobot_moveit)/launch/ros_controllers.launch"/>

</launch>
